functions-framework-api/src/main/java/com/google/cloud/functions/Context.java

```diff
@@ -19,16 +19,24 @@
 
 /** An interface for event function context. */
 public interface Context {
-  /** Returns event ID. */
+  /**
+   * @return event ID
```

Comment:
Google Java Style requires there to be at least one sentence before any `@` tags. It's fine, if redundant, to duplicate the text:

```
/**
 * Returns event ID.
 *
 * @return event ID.
 */
```

We could also possibly suppress the javadoc warnings? The remaining ones are pretty stupid.

Reply:
Fixed Javadoc sentences.

Not seeing other warnings.

```
 mvn install
[INFO] Scanning for projects...
[INFO] 
[INFO] ---------< com.google.cloud.functions:functions-framework-api >---------
[INFO] Building functions-framework-api 1.0.3-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[INFO] 
[INFO] --- maven-enforcer-plugin:1.2:enforce (enforce-maven) @ functions-framework-api ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ functions-framework-api ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /Users/timmerman/Documents/github/googlecloudplatform/functions-framework-java/functions-framework-api/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.0:compile (default-compile) @ functions-framework-api ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 8 source files to /Users/timmerman/Documents/github/googlecloudplatform/functions-framework-java/functions-framework-api/target/classes
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ functions-framework-api ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /Users/timmerman/Documents/github/googlecloudplatform/functions-framework-java/functions-framework-api/src/test/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.8.0:testCompile (default-testCompile) @ functions-framework-api ---
[INFO] No sources to compile
[INFO] 
[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ functions-framework-api ---
[INFO] No tests to run.
[INFO] 
[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ functions-framework-api ---
[INFO] Building jar: /Users/timmerman/Documents/github/googlecloudplatform/functions-framework-java/functions-framework-api/target/functions-framework-api-1.0.3-SNAPSHOT.jar
[INFO] 
[INFO] >>> maven-source-plugin:3.2.1:jar (attach-sources) > generate-sources @ functions-framework-api >>>
[INFO] 
[INFO] --- maven-enforcer-plugin:1.2:enforce (enforce-maven) @ functions-framework-api ---
[INFO] 
[INFO] <<< maven-source-plugin:3.2.1:jar (attach-sources) < generate-sources @ functions-framework-api <<<
[INFO] 
[INFO] 
[INFO] --- maven-source-plugin:3.2.1:jar (attach-sources) @ functions-framework-api ---
[INFO] Building jar: /Users/timmerman/Documents/github/googlecloudplatform/functions-framework-java/functions-framework-api/target/functions-framework-api-1.0.3-SNAPSHOT-sources.jar
[INFO] 
[INFO] --- maven-javadoc-plugin:3.1.0:jar (attach-docs) @ functions-framework-api ---
[INFO] Building jar: /Users/timmerman/Documents/github/googlecloudplatform/functions-framework-java/functions-framework-api/target/functions-framework-api-1.0.3-SNAPSHOT-javadoc.jar
[INFO] 
[INFO] --- maven-install-plugin:2.4:install (default-install) @ functions-framework-api ---
[INFO] Installing /Users/timmerman/Documents/github/googlecloudplatform/functions-framework-java/functions-framework-api/target/functions-framework-api-1.0.3-SNAPSHOT.jar to /Users/timmerman/.m2/repository/com/google/cloud/functions/functions-framework-api/1.0.3-SNAPSHOT/functions-framework-api-1.0.3-SNAPSHOT.jar
[INFO] Installing /Users/timmerman/Documents/github/googlecloudplatform/functions-framework-java/functions-framework-api/pom.xml to /Users/timmerman/.m2/repository/com/google/cloud/functions/functions-framework-api/1.0.3-SNAPSHOT/functions-framework-api-1.0.3-SNAPSHOT.pom
[INFO] Installing /Users/timmerman/Documents/github/googlecloudplatform/functions-framework-java/functions-framework-api/target/functions-framework-api-1.0.3-SNAPSHOT-sources.jar to /Users/timmerman/.m2/repository/com/google/cloud/functions/functions-framework-api/1.0.3-SNAPSHOT/functions-framework-api-1.0.3-SNAPSHOT-sources.jar
[INFO] Installing /Users/timmerman/Documents/github/googlecloudplatform/functions-framework-java/functions-framework-api/target/functions-framework-api-1.0.3-SNAPSHOT-javadoc.jar to /Users/timmerman/.m2/repository/com/google/cloud/functions/functions-framework-api/1.0.3-SNAPSHOT/functions-framework-api-1.0.3-SNAPSHOT-javadoc.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  4.911 s
[INFO] Finished at: 2020-09-24T12:17:12-05:00
[INFO] ------------------------------------------------------------------------
```

Reply:
Yeah, I already fixed the other warnings in #33 but it was annoying.